StatBot/Database/PostgresRepositories/PostgresStatsRepository.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StatBot.Database.PostgresRepositories
+{
+    public class PostgresStatsRepository : IRepository<Statistic>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix this implementation of 'IDisposable' to conform to the dispose pattern. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://172.21.13.172:9091/coding_rules#rule_key=csharpsquid%3AS3881)